doc/command_syntax_usage.rst

```diff
@@ -14,6 +14,7 @@ There are eight rho commands:
  * ``edit`` - to modify an existing entry
  * ``clear`` - to remove any or all entries
  * ``show`` and ``list`` - to display one or more entries
+ * ``list-facts`` - to list the facts Rho can collect
```

Comment:
I would prefer the command be named `rho fact list` which would fit the `rho <noun> <verb>` flow found in other commands.